### ISSUE TITLE
ValueError: not enough values to unpack (expected 7, got 3)

### ISSUE THREAD

**Issue by wangjianqiao111** [DataCanvasIO/DeepTables#89]
<em>Please make sure that this is a bug. </em>

**System information**
- OS Platform and Distribution (e.g., CentOS 7.6):linux
- Python version: 3.7
- DeepTables version:0.2.5
- Other Python packages(run `pip list`):

Package                      Version
---------------------------- ------------
absl-py                      2.0.0
astunparse                   1.6.3
attrs                        23.1.0
backcall                     0.2.0
bcrypt                       4.0.1
cachetools                   5.3.2
category-encoders            2.6.2
certifi                      2023.7.22
cffi                         1.15.1
charset-normalizer           3.3.0
cloudpickle                  2.2.1
cryptography                 41.0.5
dask                         2022.2.0
debugpy                      1.7.0
decorator                    5.1.1
deeptables                   0.2.5
eli5                         0.13.0
entrypoints                  0.4
flatbuffers                  1.12
fsspec                       2023.1.0
gast                         0.4.0
google-auth                  2.23.3
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
graphviz                     0.20.1
grpcio                       1.59.0
h5py                         3.8.0
hypernets                    0.3.0
idna                         3.4
importlib-metadata           6.7.0
importlib-resources          5.12.0
ipykernel                    6.16.2
ipython                      7.34.0
jedi                         0.19.1
Jinja2                       3.1.2
joblib                       1.3.2
jupyter_client               7.4.9
jupyter_core                 4.12.0
keras                        2.9.0
Keras-Preprocessing          1.1.2
libclang                     16.0.6
lightgbm                     4.1.0
locket                       1.0.0
Markdown                     3.4.4
MarkupSafe                   2.1.3
matplotlib-inline            0.1.6
nest-asyncio                 1.5.8
numpy                        1.21.6
oauthlib                     3.2.2
opt-einsum                   3.3.0
packaging                    23.2
pandas                       1.3.5
paramiko                     3.3.1
parso                        0.8.3
partd                        1.4.1
patsy                        0.5.3
pexpect                      4.8.0
pickleshare                  0.7.5
pip                          23.0.1
prettytable                  3.7.0
prompt-toolkit               3.0.39
protobuf                     3.19.6
psutil                       5.8.0
ptyprocess                   0.7.0
py4j                         0.10.9.7
pyarrow                      12.0.1
pyasn1                       0.5.0
pyasn1-modules               0.3.0
pycparser                    2.21
Pygments                     2.16.1
PyNaCl                       1.5.0
pyspark                      3.4.1
python-dateutil              2.8.2
pytz                         2023.3.post1
PyYAML                       6.0.1
pyzmq                        25.1.1
requests                     2.31.0
requests-oauthlib            1.3.1
rsa                          4.9
scikit-learn                 1.0.2
scipy                        1.7.3
setuptools                   58.1.0
six                          1.16.0
statsmodels                  0.13.5
tabulate                     0.9.0
tensorboard                  2.9.1
tensorboard-data-server      0.6.1
tensorboard-plugin-wit       1.8.1
tensorflow                   2.9.0
tensorflow-estimator         2.9.0
tensorflow-io-gcs-filesystem 0.34.0
termcolor                    2.3.0
threadpoolctl                3.1.0
toolz                        0.12.0
tornado                      6.2
tqdm                         4.66.1
traitlets                    5.9.0
typing_extensions            4.7.1
urllib3                      2.0.7
wcwidth                      0.2.8
Werkzeug                     2.2.3
wheel                        0.41.2
wrapt                        1.15.0
XlsxWriter                   3.1.9
zipp                         3.15.0

**
[LightGBM] [Info] Number of positive: 87, number of negative: 713
[LightGBM] [Info] Auto-choosing row-wise multi-threading, the overhead of testing was 0.000197 seconds.
You can set `force_row_wise=true` to remove the overhead.
And if memory is not enough, you can set `force_col_wise=true`.
[LightGBM] [Info] Total Bins 698
[LightGBM] [Info] Number of data points in the train set: 800, number of used features: 15
[LightGBM] [Info] [binary:BoostFromScore]: pavg=0.108750 -> initscore=-2.103573
[LightGBM] [Info] Start training from score -2.103573
[LightGBM] [Warning] No further splits with positive gain, best gain: -inf
[LightGBM] [Warning] No further splits with positive gain, best gain: -inf
2023-10-26 11:40:46.462244: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2023-10-26 11:40:46.462326: W tensorflow/stream_executor/cuda/cuda_driver.cc:269] failed call to cuInit: UNKNOWN ERROR (303)
2023-10-26 11:40:46.462377: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:163] no NVIDIA GPU device is present: /dev/nvidia0 does not exist
2023-10-26 11:40:46.485051: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
10-26 11:41:58 E hypernets.m.hyper_model.py 103 - run_trail failed! trail_no=1
10-26 11:41:59 E hypernets.m.hyper_model.py 105 - Traceback (most recent call last):
  File "/usr/local/python3.7.17/lib/python3.7/site-packages/hypernets/model/hyper_model.py", line 96, in _run_trial
    scores, oof, oof_scores, X_trains, y_trains, x_vals, y_vals = ret_data
ValueError: not enough values to unpack (expected 7, got 3)
**

**Describe the expected behavior**

**Standalone code to reproduce the issue**
>>> from deeptables.datasets import dsutils
>>> from deeptables.models import make_experiment
>>> from sklearn.model_selection import train_test_split
>>> from sklearn.metrics import classification_report
>>> df = dsutils.load_bank().head(1000)
>>> train_data, test_data = train_test_split(df, test_size=0.2, random_state=42)
>>> experiment = make_experiment(train_data.copy(), target='y', reward_metric='Precision', pos_label='yes', max_trials=5)
>>> estimator = experiment.run()

**Are you willing to submit PR?(Yes/No)**

**Other info / logs** 
Include any logs or source code that would be helpful to diagnose the problem. 
If including tracebacks, please include the full traceback. Large logs and files 
should be attached.

**Comment by prakhyat-sc** [comment]
Facing the same issue.

**Comment by oaksharks** [comment]
Try to downgrade `hypernets<0.3`  or use deeptables on the master branch.

**Comment by prakhyat-sc** [comment]
Thanks @oaksharks , Installing from Master Branch has resolved the issue.
